cmd/pro/provider/up.go

```diff
@@ -263,7 +263,7 @@ func createWorkspace(ctx context.Context, baseClient client.Client, log log.Logg
 			return false, err
 		}
 
-		if !isReady(workspace) {
+		if !isReady(workspace) && isRunnerReady(workspace, storagev1.BuiltinRunnerName) {
 			log.Debugf("Workspace %s is in phase %s, waiting until its ready", workspace.Name, workspace.Status.Phase)
```

Comment:
I guess this log can now mean that either workspace is not ready or runner is not ready?

Reply:
broke it up into two checks and logs 👍